TestProjects/FbxTests/ProjectSettings/ProjectVersion.txt

```diff
@@ -1,2 +1,2 @@
-m_EditorVersion: 2019.2.0a5
-m_EditorVersionWithRevision: 2019.2.0a5 (5ee2646333c0)
+m_EditorVersion: 2019.2.17f1
```

Comment:
Looks like you are using an old version of Unity. Your should use the LTS version (Long Term Support), which is 2019.4 for Unity 2019.

Not worth changing for this PR but you should launch an install at some point

Reply:
Ah, I just realized I added some extraneous files. I'll remove them now.